SixTest/stf_output/not_stf/old_converter.f

```diff
@@ -0,0 +1,41 @@
+      program old_converter
```

Comment:
A comment describing what the program does (converting non-STF format fort.90 files to ASCII), who wrote it and when would be very useful.